frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetAccessManagement/ClusterSetAccessManagement.test.tsx

```diff
@@ -0,0 +1,164 @@
+  },
+  subjects: [
+    {
+      kind: 'Group',
+      apiGroup: 'rbac.authorization.k8s.io',
+      name: mockGroup2!.metadata.name!,
+    },
+  ],
+  roleRef: {
+    apiGroup: 'rbac.authorization.k8s.io',
+    kind: 'ClusterRole',
+    name: `open-cluster-management:managedclusterset:view:${mockManagedClusterSet!.metadata.name!}`,
+  },
+}
+
+const Component = (props: { isGlobal?: boolean }) => (
+  <RecoilRoot
+    initializeState={(snapshot) => {
+      snapshot.set(managedClusterSetsState, [props.isGlobal ? mockGlobalManagedClusterSet : mockManagedClusterSet])
+    }}
+  >
+    <MemoryRouter
+      initialEntries={[
+        generatePath(NavigationPath.clusterSetDetails, {
+          id: props.isGlobal ? mockGlobalManagedClusterSet.metadata.name! : mockManagedClusterSet.metadata.name!,
+        }),
+      ]}
+    >
+      <Routes>
+        <Route path={`${NavigationPath.clusters}/*`} element={<Clusters />} />
+      </Routes>
+    </MemoryRouter>
+  </RecoilRoot>
+)
+
+describe('Cluster Sets User management', () => {
+  const renderComponent = async (mockUser: User, mockGroup: Group, mockClusterRoleBinding: ClusterRoleBinding) => {
+    const getNocks = [
+      nockClusterList(mockUser, [mockUser]),
+      nockClusterList(mockGroup, [mockGroup]),
+      nockClusterList(mockClusterRoleBinding, [mockClusterRoleBinding]),
+    ]
+    nockIgnoreRBAC()
+    nockIgnoreApiPaths()
+    render(<Component />)
+    await waitForNocks(getNocks)
+    await waitForText(mockManagedClusterSet.metadata.name!, true)
+    await waitForText('Details')
+    await waitForText('User management', true)
+    await clickByText('User management', 0)
+    await waitForTestId('addUserGroup')
+    await waitForText(mockGroup.metadata.name!)
+  }
+
+  const verifyModalContent = async (expectedText: string) => {
+    const button = screen.getByRole('button', { name: 'View users in group' })
+    userEvent.click(button)
+    const modal = await waitFor(() => screen.getByRole('dialog'))
+
+    expect(screen.getByLabelText('Close')).toBeInTheDocument()
+    expect(screen.getByText('Users in group')).toBeInTheDocument()
+    expect(within(modal).getByText(expectedText)).toBeInTheDocument()
+  }
+
+  test.each`
+    testCase                                                                          | mockUser     | mockGroup               | mockClusterRoleBinding     | expectedText
+    ${'user name in the AcmLabel when users are available in the group'}              | ${mockUser1} | ${mockGroupWithUsers}   | ${mockClusterRoleBinding1} | ${mockUser1.metadata.name}
+    ${'"No users in group" in the AcmLabel when no users are available in the group'} | ${mockUser2} | ${mockGroupWithNoUsers} | ${mockClusterRoleBinding2} | ${'No users in group'}
+  `('displays $testCase', async ({ mockUser, mockGroup, mockClusterRoleBinding, expectedText }) => {
+    await renderComponent(mockUser, mockGroup, mockClusterRoleBinding)
+    await verifyModalContent(expectedText)
+  })
+})
```

Comment:
I think this describe block needs at least one `it` block...
Check out the SonarCloud [scan result](https://sonarcloud.io/project/issues?pullRequest=3792&open=AZF-0G8hLa5YzSEVsEk2&id=open-cluster-management_console).